operator/cmd/mesh/profile-common.go

```diff
@@ -48,11 +62,24 @@ import (
 // In step 3, the remaining fields in the same user overlay are applied on the resulting profile base.
 // The force flag causes validation errors not to abort but only emit log/console warnings.
 func GenerateConfig(inFilenames []string, setOverlayYAML string, force bool, kubeConfig *rest.Config,
-	l clog.Logger) (string, *v1alpha1.IstioOperatorSpec, error) {
+	l clog.Logger, useClusterIfAvailable bool) (string, *v1alpha1.IstioOperatorSpec, error) {
 
-	fy, profile, err := readYamlProfle(inFilenames, setOverlayYAML, force, l)
-	if err != nil {
-		return "", nil, err
+	var err error
+	var fy, profile string
+	if useClusterIfAvailable && len(inFilenames) == 0 {
+		fy, err = overlayedIOPFromCluster("istio-system", setOverlayYAML, kubeConfig)
+		if err != nil {
+			l.Print("No previous install; using built-in profile\n")
```

Comment:
this message is not quite right, since the profile could some from overlay file, set flag or external charts (or combination of those). suggest saying just "No previous install detected, using passed-in settings only." or something like that.

Reply:
Can we instead output a message only when we do detect in settings? this might confuse new users

Reply:
@howardjohn can you explain your message preference?
@ostromart How about we do the opposite, and say "Using profile <x>", only if we find one?

Reply:
sure, that works too.